src/test/java/com/ohdab/member/service/GetTeacherListServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package com.ohdab.member.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.ohdab.member.domain.Authority;
+import com.ohdab.member.domain.Member;
+import com.ohdab.member.domain.memberinfo.MemberInfo;
+import com.ohdab.member.repository.MemberRepository;
+import com.ohdab.member.service.dto.MemberDto;
+import com.ohdab.member.service.mapper.ServiceMemberMapper;
+import com.ohdab.member.service.usecase.GetTeacherListUsecase;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {GetTeacherListService.class, ServiceMemberMapper.class})
```

Comment:
```suggestion
@ContextConfiguration(classes = {GetTeacherListService.class})
```
mappper를 필드에서 주입받지 않고 있어서 컨테이너에 등록 안해줘도 될거같음

Reply:
아 다른거랑 헷갈림 화긴